README.md

```diff
@@ -1 +1,39 @@
-# Hello-world
\ No newline at end of file
+# Hello-world
+
+## Built With
+HTML
+CSS
+Linters
+Git/Github
+Live Demo Link
+
+### Getting Started
+Live Demo Link  
+
+#### Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+## Prerequisites
+
+Setup GitHub repository
+Install git
```

Comment:
- [ ] Kindly add a descriptive README file to your project. please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

Reply:
Thank you , Sure, will do as advised